HostWormsGame.ps1

```diff
@@ -0,0 +1,30 @@
+$yourSecretSlackToken = 'ENTER YOUR SECRET SLACK TOKEN' # Get from https://api.slack.com/web#authentication
+
+$channel = '#games-worms'
+$optionsRepoUrl = 'https://api.github.com/repos/TheEadie/WormsLeague/releases/latest'
+$installDirPath = 'C:\Program Files (x86)\Steam\steamapps\common\Worms Armageddon\'
+$schemesDirPath = Join-Path $installDirPath '\User\Schemes\'
+
+function Send-Slack() {
+	$ip = (ipconfig | grep "10.120" -m 1).Substring(39)
```

Comment:
Don't make it download grep, use native Powershell. I'm pretty sure .net has a way to get your IP address!